docs/core/entity/calendar.md

```diff
@@ -23,6 +26,16 @@ Properties should always only return information from memory and not do I/O (lik
 | `STATE_ON`  | The calendar has an active event.           |
 | `STATE_OFF` | The calendar does not have an active event. |
 
+## Supported Features
+
+Supported features are defined by using values in the `CalendarEntityFeature` enum
+and are combined using the bitwise or (`|`) operator.
+
+| Value               | Description                                                        |
+| ------------------- | ------------------------------------------------------------------ |
+| `MUTATION`      | Entity implements the methods to allow creation and deletion of events.  |
```

Comment:
Would it be safer to distinguish between creation and deletion?

Reply:
Done. Updated the Core PR and Frontend with this split into two features.